src/conic.jl

```diff
@@ -592,6 +601,10 @@ function addPrimalCuttingPlanes!(m, mip_model, separator, cb)
                         new_rhs += V[i] * separator[I[i]]
                     end
                     #new_rhs = (abs(new_rhs) < 1e-9 ? 0.0 : new_rhs)
+                    viol = vecdot(V, mip_solution[I]) - (initial_query ? 0.0 : getValue(m.mip_t[k][j-1])) - new_rhs
```

Comment:
Its kind of checking the max distance of the mip_solution from the separating inequalities, is it unnecessary??